### PR TITLE
documentation: ZENKO-1899_Fix_general_problems_in_documentation

### PR DESCRIPTION
**Pull request by wabernat**
This PR fixes a lot of typos, faulty indentations, and minor formatting issues.

We need it because every defect is another way for us to be judged wanting.  

This fixes #ZENKO-1899

**Special notes for your reviewers**:

I apologize for the complexity of this commit. With no internet connectivity 
at home Friday, the best way I could spend my time was a general-purpose
edit. Unfortunately, I was also unable to start a new ticket for each issue.

This is a "sweep" of minor problems in the Zenko documentation, mostly
issues like typos, poorly formatted tables, poorly indented lines, lines
with a space at the end, no newline at end of file, etc.

I changed the suspiciously S3C-ish heading, "Zenko Connector API
Knowledge Primer" to "Zenko API Primer"

I changed some of the headings from gerunds ("Deleting files") to
the imperative ("Delete Files", e.g.), a simpler and more muscular
construction.

I moved uninstall instructions out from under an inapt header to
its own topic.

I changed all instances of "options.yml" to "options.yaml"

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e reset

File: docs/docsource/operation/Architecture/CloudServer.rst [hunk 1]
```diff
@@ -94,28 +94,28 @@ use cases.
...
 
 .. tabularcolumns:: X{0.45\textwidth}X{0.45\textwidth}
-.. table:: NodeJS Process General Metrics
+.. table:: Node.js Process General Metrics
 
    +-----------------------------------------------+---------------------------------------------------------+
    | Metric                                        | Description                                             |
    +===============================================+=========================================================+
-   | nodejs\_version\_info                         | NodeJS Version info                                     |
+   | nodejs\_version\_info                         | Node.js Version info                                    |
```
Comment (JianqinWang) [inline docs/docsource/operation/Architecture/CloudServer.rst]: `Version` probably doesn't need to be capitalized, but not an important detail.

File: docs/docsource/operation/Orbit_UI/Advanced_Workflows/Advanced_Workflows.rst [hunk 2]
```diff
@@ -11,7 +11,7 @@ You can use Orbit to manage the following advanced workflows:
    :maxdepth: 1
    :hidden:
    
-   Setting Up Replication <Setting_Up_CRR>
+   Setting Up Replication <set_up_crr>
```
Comment (JianqinWang) [inline docs/docsource/operation/Orbit_UI/Advanced_Workflows/Advanced_Workflows.rst]: Are we changing this to `Set Up Replication` as well?

File: docs/docsource/operation/Architecture/CloudServer.rst [hunk 3]
```diff
@@ -6,7 +6,7 @@ handling the Amazon S3 protocols.
 
 By providing a free-standing implementation of the S3 API, CloudServer
 offers developers the freedom to build S3 apps and run them either
-on-premises, in the AWS public cloud, or both—with no code changes.
+on-premises, in the AWS public cloud, or both—-with no code changes.
```
Comment (ssalaues) [inline docs/docsource/operation/Architecture/CloudServer.rst]: em dash

File: docs/docsource/operation/Architecture/CloudServer.rst [hunk 4]
```diff
@@ -94,28 +94,28 @@ use cases.
 
 -  **Healthcheck**
 
-   Currently, a liveness probe calls /\_/healthcheck/deep. Services that
-   expose readiness can also get a readiness probe.
+   Currently, a liveness probe calls ``/\_/healthcheck/deep``. Services
```
Comment (ssalaues) [inline docs/docsource/operation/Architecture/CloudServer.rst]: String literal

File: docs/docsource/operation/Introduction/Conceptual_Framework.rst [hunk 5]
```diff
@@ -7,15 +7,15 @@ control, manage, replicate, and visualize data in a multi-cloud context.
 Fundamental building blocks of Zenko’s replication framework are the
 bucket, the location, and the cloud service. End users (people or
 services) store data as files in buckets. Administrators are free to
-configure and name buckets as they please—to the admin, buckets are
+configure and name buckets as they please—-to the admin, buckets are
```
Comment (ssalaues) [inline docs/docsource/operation/Introduction/Conceptual_Framework.rst]: em dash

File: docs/docsource/operation/Introduction/Conceptual_Framework.rst [hunk 6]
```diff
@@ -7,15 +7,15 @@ control, manage, replicate, and visualize data in a multi-cloud context.
...
 essentially tenants aggregated in a cloud storage deployment. From an
 end user’s perspective, a bucket simply appears as a networked storage
 location, for example, “accounting,” “bioinformatics-lab,”
 “daily-video-backup,” or any other organizational unit that makes
 sense.
 
-Zenko fetches buckets from one location—a Scality RING private cloud,
-for example—and replicates them to designated cloud storage locations on
+Zenko fetches buckets from one location—-a Scality RING private cloud,
```
Comment (ssalaues) [inline docs/docsource/operation/Introduction/Conceptual_Framework.rst]: Some more em dashes

File: docs/docsource/operation/Architecture/CloudServer.rst (hunk 4 above)
Comment (wabernat) [reply docs/docsource/operation/Architecture/CloudServer.rst]: fixed

File: docs/docsource/operation/Architecture/CloudServer.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/operation/Architecture/CloudServer.rst]: fair spot.

File: docs/docsource/operation/Introduction/Conceptual_Framework.rst (hunk 5 above)
Comment (wabernat) [reply docs/docsource/operation/Introduction/Conceptual_Framework.rst]: fixed

File: docs/docsource/operation/Introduction/Conceptual_Framework.rst (hunk 6 above)
Comment (wabernat) [reply docs/docsource/operation/Introduction/Conceptual_Framework.rst]: fixed

**Review by dora-korpar — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

File: docs/docsource/operation/Orbit_UI/Advanced_Workflows/Advanced_Workflows.rst (hunk 2 above)
Comment (wabernat) [reply docs/docsource/operation/Orbit_UI/Advanced_Workflows/Advanced_Workflows.rst]: Done.

**Comment by wabernat** [comment]
@bert-e approve

**Comment by wabernat** [comment]
@bert-e reset

**Pull request merged**